packages/config/config/devices/0x0234/matrix_zba7140.json

```
@@ -0,0 +1,634 @@
+					"label": "LEDs indicates the current level of the battery, when the device is in",
+					"value": 1
+				},
+				{
+					"label": "LED colour is set by configuration parameter 5.",
+					"value": 2
+				}
+			]
+		},
+		"6": {
+			"label": "Associations groups, transmission when included secure.",
+			"description": "This parameter specifies if commands are transmitted as a secure message for each of the association groups.",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "All messages in all groups are sent as insecure.",
+					"value": 0
+				},
+				{
+					"label": "Messages in association group 2 are sent as secure.",
+					"value": 1
+				},
+				{
+					"label": "Messages in association group 3 are sent as secure.",
+					"value": 2
+				},
+				{
+					"label": "Messages in association group 4 are sent as secure.",
+					"value": 4
+				},
+				{
+					"label": "Messages in association group 5 are sent as secure.",
+					"value": 8
+				},
+				{
+					"label": "Messages in association group 6 are sent as secure.",
+					"value": 16
+				},
+				{
+					"label": "Messages in association group 7 are sent as secure.",
+					"value": 32
+				},
+				{
+					"label": "Messages in association group 8 are sent as secure.",
+					"value": 64
+				},
+				{
+					"label": "Messages in association group 9 are sent as secure.",
+					"value": 128
+				},
+				{
+					"label": "All messages in all groups  are sent as secure. (Default)",
+					"value": 255
+				}
+			]
+		},
+		"7": {
+			"label": "Pushbutton 1 functionality.",
+			"description": "This parameter specifies the functionality of pushbutton 1.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"10": {
+			"label": "Pushbutton 2 functionality.",
+			"description": "This parameter specifies the functionality of pushbutton 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"13": {
+			"label": "Pushbutton 3 functionality.",
+			"description": "This parameter specifies the functionality of pushbutton 3.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"16": {
+			"label": "Pushbutton 4 functionality.",
+			"description": "This parameter specifies the functionality of pushbutton 4.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5[0xff]": {
+			"label": "LEDs colour indication, Byte 1",
+			"description": "Specifies the level for the red colour. (Default is 255)",
```

```suggestion
			"label": "LED colour: Red level",
```
similar for the next two